test/config/TestConfig.cpp

```diff
@@ -14,14 +14,22 @@ using namespace Aws::Crt;
 using namespace Aws::Iot::DeviceClient;
 using namespace Aws::Iot::DeviceClient::Util;
 
+TEST(Config, CreateTempFile)
+{
+    string filePath = "/tmp/test";
```

Comment:
Consider saving the file path in some variable if possible since it will be used everywhere in this file and also that way it would be easier to update the file path.